packages/theme/src/components/tabs.ts

```diff
@@ -122,6 +123,10 @@ const variantEnclosed: PartsStyleFunction<typeof parts> = (props) => {
         borderColor: "inherit",
         borderBottomColor: mode(`white`, `gray.800`)(props),
       },
+      _disabled: {
+        cursor: "not-allowed",
+        opacity: 0.4,
+      },
```

Comment:
Great work here. Thank you!

How about we add this to the base style of the tab instead of duplicating it in each variant? I think we should update the `baseStyleTab` styles instead.

Reply:
Good point, I will get this updated today. Thanks!